sound/soc/intel/boards/sof_sdw_rt722_sdca.c

```diff
@@ -33,7 +33,7 @@ int rt722_spk_rtd_init(struct snd_soc_pcm_runtime *rtd)
 	int ret;
 
 	card->components = devm_kasprintf(card->dev, GFP_KERNEL,
-					  "%s spk:rt722",
+					  "%s spk:rt722-spk",
```

Comment:
This is the `spk:` tag, whatever is after it _is_ a speaker? Why do you need to spell it out?

Reply:
> This is the `spk:` tag, whatever is after it _is_ a speaker? Why do you need to spell it out?

UCM get the conf name from the string after the tag. And the purpose is to use different conf files for different devices. For example, rt722-spk.conf for speaker configuration, rt722-mic.conf for mic configuration. However, the direction was changed to use a single rt722.conf for all device configurations. I thought I have closed this PR, don't know why it is still opne.